### PR TITLE
Add XCCDF results into user manual

### PR DESCRIPTION
**Pull request by jan-cerny**
Users don't know what the results of rule evaluation mean. Although
it's specified in XCCDF specification, it's easier to find it
described in user's manual. I have mostly copied the table 26 from
https://csrc.nist.gov/CSRC/media/Publications/nistir/7275/rev-4/final/documents/nistir-7275r4_updated-march-2012_clean.pdf
and have added some OpenSCAP specific examples.

### CODE REVIEW

File: docs/manual/manual.adoc [hunk 1]
```diff
@@ -445,6 +445,41 @@ Ident   CCE-3967-7
 Result  pass
 ----
 
+The meaning of results is defined by https://csrc.nist.gov/CSRC/media/Publications/nistir/7275/rev-4/final/documents/nistir-7275r4_updated-march-2012_clean.pdf[XCCDF Specification].
+This table lists the possible results of a single rule:
+
+.XCCDF results
+|===
+|Result |Description
```
Comment (yuumasato) [inline docs/manual/manual.adoc]: Does it make sense to separate the description from the example into a new column?

File: docs/manual/manual.adoc [hunk 2]
```diff
@@ -445,6 +445,41 @@ Ident   CCE-3967-7
...
+|The checking engine could not complete the evaluation, therefore the status of the target’s compliance with the rule is not certain. This could happen, for example, if a testing tool was run with insufficient privileges and could not gather all of the necessary information.
+
+|unknown
+|The testing tool encountered some problem and the result is unknown. For example, a result of ‘unknown’ might be given if the testing tool was unable to interpret the output of the checking engine (the output has no meaning to the testing tool)
+
+|notapplicable
+|The rule was not applicable to the target of the test. For example, the rule might have been specific to a different version of the target OS, or it might have been a test against a platform feature that was not installed.
+
+|notchecked
+|The rule was not evaluated by the checking engine. This status is designed for rules that have no <xccdf:check> elements or that correspond to an unsupported checking system. It may also correspond to a status returned by a checking engine if the checking engine does not support the indicated check code. For example, the rule does not reference any OVAL check.
```
Comment (yuumasato) [inline docs/manual/manual.adoc]: Improve wording here. The example seems linked to the last item of description.

File: docs/manual/manual.adoc (hunk 1 above)
Comment (jan-cerny) [reply docs/manual/manual.adoc]: Yes, it can probably improve the orientation for the reader.

File: docs/manual/manual.adoc (hunk 2 above)
Comment (jan-cerny) [reply docs/manual/manual.adoc]: ok

**Comment by jan-cerny** [comment]
@jan-cerny I have improved it a little.

**Pull request merged**